src/autodiff_model.jl

```diff
@@ -156,6 +168,21 @@ function hess(nlp :: ADNLPModel, x :: AbstractVector; obj_weight :: Real = one(e
   return tril(Hx)
 end
 
+function hess_structure(nlp :: ADNLPModel)
+  n = nlp.meta.nvar
+  I = ((i,j) for i = 1:n, j = 1:n if i ≥ j)
+  return (getindex.(I, 1), getindex.(I, 2))
+end
+
+function hess_coord!(nlp :: ADNLPModel, x :: AbstractVector, rows :: AbstractVector{Int}, cols :: AbstractVector{Int}, vals :: AbstractVector; obj_weight :: Real = one(eltype(x)), y :: AbstractVector = eltype(x)[])
```

Comment:
Here `rows` is parametrized but in `jac_coord` it isn't.

Reply:
Thanks. Updated.